### PR TITLE
App Loader script

### PR DESCRIPTION
**Pull request by zfox23**
See README for testing instructions.

You can use this release for testing:
https://hifi-content.s3.amazonaws.com/Experiences/Releases/usefulUtilities/appLoader/2019-02-14_10-45-00/appLoader.js

### CODE REVIEW

File: usefulUtilities/appLoader/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- App Loader will load scripts defined as hardcoded URLs in the JavaScript, AND will load scripts defined in the `userData` property of the attached entity.
+- When the user leaves the domain (or otherwise unloads the script attached to the entity), App Loader will stop client scripts that it loaded. App Loader will NOT stop client scripts that it did not load (i.e. those that were previously running).
+
+## Setup
+1. Add an entity to your domain.
+    - A Box primitive would work perfectly.
+    - Note that anyone who can modify the `userData` of this entity will be able to control which scripts other users load!
+    - A good place for this entity is right under the floor near your domain's spawn point. You want to ensure that all visitors to your domain load this entity.
+2. Add the below `userData` object to the attached entity's `userData`
+    1. Fill in the `apps` array in the `userData` by adding URLs to scripts that you want users to load.
```
Comment (MiladNazeri) [inline usefulUtilities/appLoader/README.md]: Did you mean to use 1. here?  It looks a little strange in the view file

File: usefulUtilities/appLoader/appLoader.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+//
+//  appLoader.js
+//
+//  Created by Zach Fox on 2019-02-14
```
Comment (MiladNazeri) [inline usefulUtilities/appLoader/appLoader.js]: ❤️

File: usefulUtilities/appLoader/README.md (hunk 1 above)
Comment (zfox23) [reply usefulUtilities/appLoader/README.md]: Indeed I did! Nested numbered lists render as (i) or (a) (i forget which one) in Markdown (at least on GitHub).

File: usefulUtilities/appLoader/appLoader.js (hunk 2 above)
Comment (zfox23) [reply usefulUtilities/appLoader/appLoader.js]: ♥

File: usefulUtilities/appLoader/README.md (hunk 1 above)
Comment (MiladNazeri) [reply usefulUtilities/appLoader/README.md]: got it, yeah it listed as (i).

**Pull request merged**